covid_graphs/covid_graphs/scripts/country_dashboard.py

```diff
@@ -5,90 +5,136 @@
     ]
     country_graphs.sort(key=lambda graph: graph.short_name)
+    return country_graphs
+
+
+def create_dashboard(
+    data_dir: Path, server: Flask,
+):
+    # TODO(miskosz): Don't use print.
+    print("Creating dashboard for prediction graphs.")
+    prediction_events = prediction_db.get_prediction_events()
+    prediction_events.sort(key=lambda event: event.date, reverse=True)
+    prediction_event_from_name = {
+        prediction_event.name: prediction_event for prediction_event in prediction_events
+    }
+
+    # TODO: parsing the proto can fail
+    reports = [
+        CountryReport(data_dir / f"{country_short_name}.data", country_short_name)
+        for country_short_name in prediction_db.get_countries()
+        if (data_dir / f"{country_short_name}.data").is_file()
+    ]
+    graph_dict = {
+        prediction_event.name: create_graphs(reports, prediction_event)
+        for prediction_event in prediction_db.get_prediction_events()
+    }
 
     app = dash.Dash(
-        name=f'COVID-19 predictions {prediction_event.name}',
-        url_base_pathname=f'/covid19/{prediction_event.name}/',
+        name=f"COVID-19 predictions",
+        url_base_pathname=f"/covid19/predictions/",
         server=server,
         external_scripts=[
-            'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/MathJax.js?config=TeX-MML-AM_CHTML'
+            "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/MathJax.js?config=TeX-MML-AM_CHTML"
         ],
     )
-    app.title = 'COVID-19 predictions of Boďová and Kollár'
+    app.title = "COVID-19 predictions of Boďová and Kollár"
 
     content = _get_header_content(app.title)
 
-    next_day = prediction_event.date + timedelta(days=1)
-    content += [
-        html.Hr(),
-        html.H1(f"{next_day.strftime('%B %d')} predictions")
-    ]
+    content += [html.Hr(), html.H1(id="graph-title")]
     buttons = [
-        dcc.Dropdown(id='country-short-name',
-                     options=[
-                         dict(label=graph.long_name, value=graph.short_name)
-                         for graph in country_graphs
-                     ],
-                     value='Italy',
-                     style={'width': '30%'}),
-        dcc.RadioItems(id='graph-type',
-                       options=[{
-                           'label': graph_type.value,
-                           'value': graph_type.name,
-                       } for graph_type in [GraphType.Normal, GraphType.SemiLog, GraphType.LogLog]],
-                       value='Normal',
-                       labelStyle={'display': 'inline-block'})
+        dcc.Dropdown(
+            id="prediction-event",
+            options=[
+                dict(label=(event.date + timedelta(days=1)).strftime("%B %d"), value=event.name)
+                for event in prediction_events
+                if event != OTHER
+            ],
+            value=BK_20200411.name,
+            style={"width": "40%"},
```

Comment:
We should add a css for the whole page to make it nicer btw :)

Reply:
Yeah, but that's for version 2.1 :)